apple/main.py

```diff
@@ -196,7 +219,7 @@ def main():
             start,
             end,
             extra_meta={
-                "episode": episode_id,
+                "episodeid": episode_id,
```

Comment:
the `api` is expecting `episode` in the json. see https://github.com/openpodcast/api/blob/8b020f597ff35e94049da427a11cf5d530640a54/fixtures/appleEpisodeDetails.json#L5-L9

Reply:
I'm getting

```
2022-12-11 21:06:06.901 | INFO     | __main__:fetch_and_capture:80 - Fetching episodeDetails
2022-12-11 21:06:09.874 | INFO     | __main__:fetch_and_capture:112 - episodeDetails: must have required property 'episodeid'
```

😢

Reply:
Looks like we define `episodeid` in the schema here:

https://github.com/openpodcast/api/blob/6685949b6fdf6535342eaf6f6817c70e898a8ff7/src/schema/apple/episodeDetails.json

but we check for `episode` in the endpoint:

https://github.com/openpodcast/api/blob/6685949b6fdf6535342eaf6f6817c70e898a8ff7/src/api/connectors/AppleConnector.ts#L47-L49

Is that correct? 🤔

Reply:
My guess is that it would start working if we change `episodeid` to `episode` in the schema. Open for alternative suggestions, though.

Reply:
We resolved the issue with https://github.com/openpodcast/api/pull/34.